test/integration/ThreeChainEndToEnd.t.sol

```diff
@@ -124,7 +130,7 @@ contract ThreeChainEndToEndDeployment is EndToEndFlows {
 
         vm.prank(INVESTOR_A);
         sB.spoke.crosschainTransferShares{value: GAS}(
-            sC.centrifugeId, POOL_A, SC_1, INVESTOR_A.toBytes32(), amount, SHARE_HOOK_GAS
+            sC.centrifugeId, POOL_A, SC_1, INVESTOR_A.toBytes32(), amount, 0, SHARE_HOOK_GAS
```

Comment:
NIT. I would rename SHARE_HOOK_GAS and I would use it instead of 0 for reference:

```suggestion
            sC.centrifugeId, POOL_A, SC_1, INVESTOR_A.toBytes32(), amount, HOOG_GAS, HOOK_GAS
```